.storybook/data/migrated-components.json

```diff
@@ -0,0 +1,317 @@
+      "url": "form"
+    },
+    {
+      "name": "helptext",
+      "title": "Help text",
+      "url": "help-text"
+    },
+    {
+      "name": "icon",
+      "title": "Icon",
+      "url": "icon"
+    },
+    {
+      "name": "illustratedmessage",
+      "title": "Illustrated message",
+      "url": "illustrated-message"
+    },
+    {
+      "name": "infieldbutton",
+      "title": "In-field button",
+      "url": "in-field-button"
+    },
+    {
+      "name": "infieldprogresscircle",
+      "title": "In-field progress circle",
+      "url": "in-field-progress-circle"
+    },
+    {
+      "name": "inlinealert",
+      "title": "In-line alert",
+      "url": "in-line-alert"
+    },
+    {
+      "name": "link",
+      "title": "Link",
+      "url": "link"
+    },
+    {
+      "name": "menu",
+      "title": "Menu",
+      "url": "menu"
+    },
+    {
+      "name": "meter",
+      "title": "Meter",
+      "url": "meter"
+    },
+    {
+      "name": "opacitycheckerboard",
+      "title": "Opacity checkerboard",
+      "url": "opacity-checkerboard"
+    },
+    {
+      "name": "pagination",
+      "title": "Pagination",
+      "url": "pagination"
+    },
+    {
+      "name": "picker",
+      "title": "Picker",
+      "url": "picker"
+    },
+    {
+      "name": "pickerbutton",
+      "title": "Picker button",
+      "url": "picker-button"
+    },
+    {
+      "name": "popover",
+      "title": "Popover",
+      "url": "popover"
+    },
+    {
+      "name": "progressbar",
+      "title": "Progress bar",
+      "url": "progress-bar"
+    },
+    {
+      "name": "progresscircle",
+      "title": "Progress circle",
+      "url": "progress-circle"
+    },
+    {
+      "name": "radio",
+      "title": "Radio",
+      "url": "radio"
+    },
+    {
+      "name": "rating",
+      "title": "Rating",
+      "url": "rating"
+    },
+    {
+      "name": "search",
+      "title": "Search field",
+      "url": "search-field"
+    },
+    {
+      "name": "slider",
+      "title": "Slider",
+      "url": "slider"
+    },
+    {
+      "name": "statuslight",
+      "title": "Status light",
+      "url": "status-light"
+    },
+    {
+      "name": "stepper",
+      "title": "Number field",
+      "url": "number-field"
+    },
+    {
+      "name": "swatch",
+      "title": "Swatch",
+      "url": "swatch"
+    },
+    {
+      "name": "swatchgroup",
+      "title": "Swatch group",
+      "url": "swatch-group"
+    },
+    {
+      "name": "switch",
+      "title": "Switch",
+      "url": "switch"
+    },
+    {
+      "name": "table",
+      "title": "Table",
+      "url": "table"
+    },
+    {
+      "name": "tabs",
+      "title": "Tabs",
+      "url": "tabs"
+    },
+    {
+      "name": "tag",
+      "title": "Tag",
+      "url": "tag"
+    },
+    {
+      "name": "taggroup",
+      "title": "Tag group",
+      "url": "tag-group"
+    },
+    {
+      "name": "textfield",
+      "title": "Text area",
+      "url": "text-area"
+    },
```

Comment:
Hmmm, I was wondering why textfield didn't show up on the list for me, and it is here... just pretending to be text area instead?